src/geowombat/backends/xarray_rasterio_.py

```diff
@@ -308,14 +309,20 @@ def open_rasterio(
     coords["band"] = np.asarray(riods.indexes)
 
     # Get coordinates
+    riods_transform = riods.transform
     if riods.transform.is_rectilinear:
         # 1d coordinates
         parse = True if parse_coordinates is None else parse_coordinates
         if parse:
             nx, ny = riods.width, riods.height
+            # a, b, c, d, e, f = riods.transform
+            if riods.transform.a < 0:

```

Comment:
I'm doing this from my cell while traveling so I can't try an example. But with a negative xres it implies the array is mirrored left to right and drawn in reverse order (I think, bc that's what happens in the vertical axis if the yres is positive)

Seems like yours just abs away the xres negative value without mirroring the array. 

So not clear this will work.

Reply:
There's no rush on this one so this can sit until we are able to test it.